hl7v23/Ballerina.toml

```diff
@@ -10,4 +10,4 @@ repository="https://github.com/ballerina-platform/module-ballerinax-health.hl7v2
 [[dependency]]
 org = "ballerinax"
 name = "health.hl7v2"
-version = "1.0.1"
+version = "1.0.2"
```

Comment:
Shouldn't we bump the version of hl7v23 as well?

Reply:
This will not be immediately released. There are some other refactoring suggested by Bal team which will done after this. So version bump for releasing hl7v23 will come after those changes